Makefile

```diff
@@ -19,3 +31,31 @@ py:
 get_com:
 	source ${VENV_DIR}/bin/activate; \
 	python scripts/download_from_lesswrong.com.py > log.txt 2>&1
+
+book:
+	source ${VENV_DIR}/bin/activate; \
+	python scripts/html_to_docbook.py > log_book.txt 2>&1
+
+pdf: $(PDF_FILES)
+
+%.pdf: %.fo
+	fop -pdf $@ -fo $<
+
+%.fo: %$(BOOK_EXT)
+	xsltproc --xinclude \
+		--stringparam  profile.status "$(DOC_MODE)" \
+		/usr/share/xml/docbook/stylesheet/nwalsh/profiling/profile.xsl \
+		$< | \
+	xsltproc -o $@ \
+		--xinclude \
+		--stringparam  section.autolabel 1 \
+		--stringparam  section.label.includes.component.label 1 \
+		--stringparam  keep.relative.image.uris 1 \
+		--stringparam  disable-output-escaping no \
+		--stringparam  ulink.show 0 \
```

Comment:
Nit: I think it can be moved to config files, I have examples: https://github.com/worldmind/ratiocracy.book/blob/main/xsl/html-params.xsl